### ISSUE TITLE
Cannot run tests on several flavors within one build

### ISSUE THREAD

**Issue by roman-mazur** [stanfy/spoon-gradle-plugin#4]
Getting an error:
```
Caused by: java.lang.NullPointerException
at com.android.ddmlib.AndroidDebugBridge.stop(AndroidDebugBridge.java:732)
at com.android.ddmlib.AndroidDebugBridge.createBridge(AndroidDebugBridge.java:314)
```

**Comment by pvegh** [comment]
devices are still often disconnected. current adb is 1.0.32

**Comment by roman-mazur** [comment]
Fixed in spoon 1.1.7.
